### DISCUSSION TITLE
4.0 stable 버전에서의 board기능

### DISCUSSION THREAD

**Discussion by samknp** [storycraft/node-kakao#156] [Q&A] [answered]
4.0 stable release 축하합니다! 항상 감사히 쓰고 있습니다.
3버전에서는 있었던 기능인 requestPostList, commentToPost등은 4.0 stable버전에서는 아직 migration 안된거죠?
3버전에서의 getUserInfo() 오류로 인해 4.0으로 넘어가려 하는데 board, poll, post, comment 등은 검색해봐도 아직은 없네요.

**Comment by storycraft** [comment, marked as answer]
현재 4.0 버전 Loco 프로토콜 구현에 필요하지 않은 외부 웹 api는 최소한으로만 구현되어있습니다. 오픈채팅, 채팅방 게시판 api등은 시간이 걸릴 듯 합니다.
